packages/asset-swapper/src/utils/alt_mm_implementation_utils.ts

```diff
@@ -211,13 +215,43 @@ export async function returnQuoteFromAltMMAsync<ResponseT>(
         }
     }
 
-    const response = await axiosInstance.post(`${url}/quotes`, data, {
-        headers: { Authorization: `Bearer ${apiKey}` },
-        timeout: maxResponseTimeMs,
-    });
+    // let response: AxiosResponse<any>;
+    const response = await axiosInstance
+        .post(`${url}/quotes`, data, {
+            headers: { Authorization: `Bearer ${apiKey}` },
+            timeout: maxResponseTimeMs,
+        })
+        .then(resp => {
```

Comment:
Not sure if we need this `then`, can't we just call `.catch` right after `.post` ?

Reply:
nope 👍